src/main/scala/edu/ie3/simona/model/participant/PvModel.scala

```diff
@@ -417,7 +406,7 @@ final case class PvModel private (
     val omega2InRad = omega1InRad + omegaOneHour // requested hour plus 1 hour
 
     // (thetaG < 90°): sun is visible
-    // (thetaG > 90°), otherwise: sun is behind the surface  -> no direct radiation
+    // (thetaG > 90°), otherwise: sun is behind the surface  -> no direct irradiance
```

Comment:
Imho radiation is more correct here.

Reply:
Why? irradiance means rate of power received at the surface